src/main/java/frc/robot/Constants.java

```diff
@@ -12,4 +12,8 @@
  * <p>It is advised to statically import this class (or one of its inner classes) wherever the
  * constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
```

Comment:
To make constants more organized, you can also embed specific constants in a subconstants class.
For example:

```
public static final class IntakeConstants {
// Add relevant intake constants here
}
```

Reply:
Thank you for the suggestion Matteo! I have made changes accordingly.